extensions/bibmanager/src/index.tsx

```diff
@@ -46,10 +40,22 @@ export default function Command() {
     }
     loadItems();
   }, []);
+
+  useEffect(() => {
+    if (state.items.length > 0) {
+      filterList(filterFct(state.items, state.searchText));
```

Comment:
Instead of using another state, you could probably use `useMemo` to avoid another render

Reply:
Thanks @mathieudutour! Tried to do so. Do you mean like this?
At least it seems to avoid the issue that I described!

Reply:
Not really. `useMemo` returns the result of the operation inside. So instead of using a state for `filteredList`, you can just use the result of `useMemo`. Does that make sense?

Reply:
Ah... Now I get it. How about this one?